update chinese translation

yoshimo0417 opened the pull request:
-fix the chinese gibberish problem in WeiDU.log
-fix few chinese strings

yoshimo0417 committed:
update chinese translation

-fix the chinese gibberish problem in WeiDU.log
-fix few chinese strings

Roberciiik commented on Assassinations/Schinese/Setup-Assassinations.tra:
```diff
@@ -30,11 +30,13 @@
-�������ඨ����~
+�������ϣ�
+
+װ��������
```
If you applied EE-specific format of items, you should use Setup-Assassinations-EE,tra file for this. See Polish language file for reference.

Roberciiik commented on Assassinations/Assassinations.tp2:
```diff
@@ -59,7 +59,7 @@ LANGUAGE ~German (Translated by Toran)~ ~German~ ~Assassinations/German/Setup-As
 LANGUAGE ~Polish (Translated by Cuttooth)~ ~Polish~ ~Assassinations/Polish/Setup-Assassinations.tra~
 LANGUAGE ~French (Translated by Le Marquis)~ ~French~ ~Assassinations/French/Setup-Assassinations.tra~
-LANGUAGE ~Chinese (Translated by Merzbau)~ ~Schinese~ ~Assassinations/Schinese/Setup-Assassinations.tra~
+LANGUAGE ~Chinese (Translated by Merzbau)~ ~Schinese~ ~Assassinations/Schinese/Setup-%WEIDU_OS%.tra~
```
More safe option would be to write it as:
```
LANGUAGE ~Chinese (Translated by Merzbau)~ ~Schinese~ 
    ~Assassinations/Schinese/Setup-Assassinations.tra~
    ~Assassinations/Schinese/Setup-%WEIDU_OS%.tra~
```
To load setup file first and then override it with OS-dependent translations.

yoshimo0417 replied:
Thank you for your reply. I have fix that two problems you mentioned above. Please check.

yoshimo0417 committed:
fix format of item for chinese

-Restore the format of item in Setup-Assassinations.tra to origin game, using Setup-Assassinations-EE.tra to applying EE-specific format
-fixed Assassinations.tp2

yoshimo0417 committed:
Update Setup-Assassinations.tra

Roberciiik reviewed (approved):
Thanks for fixing comments. Looks good now. (I can't appove it as don't have privileges)

yoshimo0417 committed:
Updated some Chinese terms

yoshimo0417 committed:
update chinese translation and adjust file constrution

yoshimo0417 committed:
adjust the changes of fb0690f123005747ebf3d30203ff1347ac7e4154 and c3265916092942ca14bfe9bbdd44316efdecd5e8

yoshimo0417 committed:
Correct typo for Simplified Chinese

Roberciiik committed:
Merge branch 'master' into master

Roberciiik commented:
Hello @yoshimo0417, I've acidentially resolved conflict in your PR that commited to your repository's branch. However GitHub changed encoding for Setup-Assassination.tra file to UTF-8. Could you please convert your Chinese encoding for that file?

yoshimo0417 committed:
Update Setup-Assassinations.tra

yoshimo0417 commented:
> Hello @yoshimo0417, I've acidentially resolved conflict in your PR that commited to your repository's branch. However GitHub changed encoding for Setup-Assassination.tra file to UTF-8. Could you please convert your Chinese encoding for that file?

done.
